stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/list/MessageListView.kt

```diff
@@ -654,6 +657,15 @@ public class MessageListView : ConstraintLayout {
         scrollHelper.scrollToBottomButtonEnabled = scrollToBottomButtonEnabled
     }
 
+    /**
+     * Enables or disables the message editing feature.
+     *
+     * @param enabled True if editing a message is enabled, false otherwise.
+     */
+    public fun setEditMessageEnabled(enabled: Boolean) {
```

Comment:
We should decide whether we want to have one method will a boolean parameter or two separate for enabling and disabling. I've created two for deleting message option